upgrade/Stage_0_Prerequisites.md

```diff
@@ -27,23 +27,21 @@ Stage 0 has several critical procedures which prepare the environment and verify
    CSM_REL_NAME=csm-${CSM_RELEASE}
    ```
 
-1. Use `csm_rbd_tool.py` to create the pool, device, and mount the device.
+1. (`ncn-m001#`) Create and mount an `rbd` device where the CSM release tarball can be stored.
 
-    ```bash
-    source /opt/cray/csm/scripts/csm_rbd_tool/bin/activate
-    /usr/share/doc/csm/scripts/csm_rbd_tool/csm_rbd_tool.py --pool_action create --rbd_action create --target_host ncn-m001
-    deactivate
-    ```
+    1. Initialize the Python virtual environment.
 
-    **Note:** This same `rbd` device can be remapped to `ncn-m002` later in the upgrade procedure, when the CSM tarball is needed on that node.
+        Instructions needed
```

Comment:
do we want this here?

Reply:
No-- I was going to update that once I got instructions from Craig

Reply:
I think we're going to be remaking this PR. Talking it over with Craig.

Reply:
Yes, new PR forthcoming. Stay tuned.

Reply:
New PR: https://github.com/Cray-HPE/docs-csm/pull/2230